core/linker/default.h

```diff
@@ -155,4 +156,14 @@ SECTIONS
         . += STACK_GUARD_BAND;
         __uvisor_stack_end__ = .;
     } > STACK_S
+
+    .uninitialized (NOLOAD):
+    {
+        . = ALIGN(32);
+        __uninitialized_start = .;
+        *(.uninitialized)
+        KEEP(*(.keep.uninitialized))
+        . = ALIGN(32);
+        __uninitialized_end = .;
+    } > RAM_S
```

Comment:
The symbol is not exposed to the application so cannot be set, but if, for some reason, the application might guess the value's address I've verified that the address cannot be addressed (Bus Fault triggered).